core/ui/PaletteEditor.tid

```diff
@@ -0,0 +1,94 @@
+title: $:/PaletteEditor
```

Comment:
Indentation please.

Reply:
Just to be clear, this file hasn't been changed (yet), it is just moved from $:/PaletteManager

Reply:
ah ok

Reply:
I meant to add that this file needs substantial revision so I will add indentation when I do that.